src/apphosting/githubConnections.ts

```diff
@@ -166,7 +166,7 @@ async function createFullyInstalledConnection(
 
   while (conn.installationState.stage !== "COMPLETE") {
     utils.logBullet("Install the Firebase GitHub app to enable access to GitHub repositories");
-    const targetUri = conn.installationState.actionUri.replace("install_v2", "direct_install_v2");
+    const targetUri = conn.installationState.actionUri;
```

Comment:
This is a no-op for dev connect. This was a result of some cloud build repo copy pasta.